cern_access/indico_cern_access/client/index.js

```diff
@@ -8,6 +8,8 @@
 import {registerPluginComponent, registerPluginObject} from 'indico/utils/plugins';
 
 import CERNAccessSection, {formDecorator} from './CERNAccessSection';
+import './EnterPersonalDataForm';
+import './AccessCodeModal';
```

Comment:
Use snake_cased filenames for those files since they do not export react components.